medusa/subtitle_providers/legendastv.py

```diff
@@ -51,6 +51,9 @@
 #: Cache key for releases
 releases_key = __name__ + ':releases|{archive_id}|{archive_name}'
 
+# Override so we can detect earlier if new shows were added to LTV
+SHOW_EXPIRATION_TIME = timedelta(hours=12).total_seconds()
```

Comment:
All subtitles from seasons premier wont be found because of this cache..

This is a critical issue IMO